.github/fabricbot.json

```diff
@@ -1,3600 +0,0 @@
-                  }
-                },
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "projectName": "Features 7.1",
-                    "columnName": "In progress"
-                  }
-                },
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "projectName": "Samples 7.1",
-                    "columnName": "In progress"
-                  }
-                }
-              ]
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Add \"In progress\" label when moved to In progress column in the project",
-        "actions": [
-          {
-            "name": "removeLabel",
-            "parameters": {
-              "label": "Completed :fire:"
-            }
-          },
-          {
-            "name": "removeLabel",
-            "parameters": {
-              "label": "needs triage :mag:"
-            }
-          },
-          {
-            "name": "addLabel",
-            "parameters": {
-              "label": "in progress :construction:"
-            }
-          }
-        ]
-      }
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "id": "dhIp1pFim",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "name": "isOpen",
-              "parameters": {}
-            },
-            {
-              "operator": "or",
-              "operands": [
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "projectName": "Bugs 7.1",
-                    "columnName": "Done"
-                  }
-                },
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "projectName": "Technical 7.1",
-                    "columnName": "Done"
-                  }
-                },
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "projectName": "Features 7.1",
-                    "columnName": "Done"
-                  }
-                },
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "projectName": "Samples 7.1",
-                    "columnName": "Done"
-                  }
-                }
-              ]
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Add \"Completed\" label when moved to Done column in the project",
```

Comment:
Project board automation is not supported in the new service, so related tasks were not ported.